apps/obs-web/src/views/Password.vue

```diff
@@ -0,0 +1,140 @@
+                    class="cursor-pointer"
+                    @click="isPwd = !isPwd"
+                />
+                </template>
+            </q-input>
+            <q-input
+                outlined
+                ref="password"
+                :type="isPwd ? 'password' : 'text'"
+                v-model="confirmedPassword"
+                label="Confirm New Password"
+                dense
+                :rules="[ val => !! val || 'Field is required',
+                          val => val.length >= 12 || 'Please use minimum 12 characters',  
+                          val => /\d/.test(val) || 'Must contain a number',
+                          val => /[a-z]/.test(val) || 'Must contain a lower case character',
+                          val => /[A-Z]/.test(val) || 'Must contain an upper case character',
+                          val => /[!#$%&()`*+,-./:;<=>?_]/.test(val) || 'Must contain a special character: !#$%&()`*+,-./:;<=>?_',
+                          val => newPassword === val || 'Must match previously entered new password']"
+            >
+                <template v-slot:append>
+                <q-icon
+                    :name="isPwd ? 'visibility_off' : 'visibility'"
+                    class="cursor-pointer"
+                    @click="isPwd = !isPwd"
+                />
+                </template>
+          </q-input>
+            <div v-if="errorMsg" style="color: red">{{errorMsg}}</div>
+            <div v-else style="color: green">{{successMsg}}</div>
+            <div>Password must contain:</div>
+          <ul>
+              <li>12 characters</li>
+              <li>No words</li>
+              <li>Upper case character</li>
+              <li>Lower case character</li>
+              <li>Numeric character</li>
+              <li>Special case character among !#$%&()`*+,-./:;<=>?_</li>
+          </ul>
+          
+            <div style="text-align: center">
+              <q-btn
+                class="full-width"
+                color="primary"
+                label="Submit"
+                dense
+                type="submit"
+                align="center"
+              />
+            </div>
+        </q-form>
+      </div>
+    </q-page-container>
+  </q-layout>
+</template>
+
+<script lang="ts">
+// https://github.com/kaorun343/vue-property-decorator
+
+import Vue from 'vue';
+import { authService } from '@boatnet/bn-auth/lib';
+import { Component, Prop } from 'vue-property-decorator';
+
+@Component
+export default class Password extends Vue {
+  @Prop({ default: undefined }) public username!: string;
+  @Prop({ default: undefined }) public SESS!: string;
+
+  private newPassword = '';
+  private confirmedPassword = '';
+  private isPwd = true;
+  private errorMsg = '';
+  private successMsg = '';
+
+  private async handleSubmit() {
+    this.errorMsg = "";
+    this.successMsg = "";
+
+    const appShortName = 'BOATNET_OBSERVER';
+    const appName = 'Research Catch';
```

Comment:
Should appName be something different for obs-web?  Or does this not matter?

Reply:
good catch, this is used to display the app name in the email, so yep I'll change it

Reply:
cool, thanks for banging this out!